app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,20 @@
 class CommentsController < ApplicationController
   load_and_authorize_resource
   before_action :set_user_and_post
+
+  def index
+    @comment = Comment.new
+    render json: @comment, only: [:Text]
+  end
+
```

Comment:
- [ ] As stated in the previous reviewer Kindly fix the error preventing the creation of a comment as well via API. When I try to create comment I got the same error as with the below screenshot 
- Creating comment 
![image](https://github.com/ismayito/Rails-blog-app/assets/93598189/b3ac997e-9552-478e-ad9f-0ef1b8364066)

- The error I got 
![image](https://github.com/ismayito/Rails-blog-app/assets/93598189/dd701016-6391-43e6-9520-d08a0d1513c5)

- The **AbstractController::DoubleRenderError** occurs when you attempt to both render and redirect in the same action. In your `CommentsController#create` action, you're calling both render` json: @comment` and `redirect_to user_post_path(@post, @user).`

- To fix this issue, you need to decide whether you want to render a JSON response or redirect to a new page. If you want to render JSON, remove the `redirect_to` line. If you want to redirect, remove the render line.

Reply:
This issue has not been resolved. You are calling the keywords render and redirect_to in the create action of the Comments controller and this is causing the `AbstractController::DoubleRenderError`. Kindly fix this error. 👍 😎